core/tests/protractor_utils/ThanksPage.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the thanks page, for use in Protractor
+ * tests.
+ */
+
+var general = require('./general.js');
+
+var ThanksPage = function(){
+  var THANKS_PAGE_URL = '/thanks';
+
+  this.get = function(){
+    return browser.get(general.SERVER_URL_PREFIX + THANKS_PAGE_URL);
```

Comment:
I took a look at the other files and found that general.SERVER_URL_PREFIX may not be needed here. Could you please try removing it and see if the tests still pass? (I'm not 100% sure myself)